src/store.rs

```diff
@@ -170,3 +173,69 @@ impl RoomKeyInfo {
         self.inner.session_id.clone()
     }
 }
+
+/// Struct containing the bundle of secrets to fully activate a new devices for
```

Comment:
nit: new devices => new device

Reply:
Fixed in: 94020b69dde69edf6963b8ad60d3af1b10ed18b1.